route-recognizer/route-recognizer.d.ts

```diff
@@ -4,22 +4,21 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "route-recognizer" {
-	
-	class RouteRecognizer<H> {
+
+	export default class RouteRecognizer<H> {
```

Comment:
this is not accurate. the module is using a bable plugin to do the transpilation, the result is still better modeled by `export =`. see https://github.com/tildeio/route-recognizer/blob/master/dist/route-recognizer.js#L753